constraintlayout/core/src/main/java/androidx/constraintlayout/core/dsl/Ref.java

```diff
@@ -125,11 +125,12 @@ public void setPostMargin(float postMargin) {
      * @param obj object to be parsed
      * @return a number
      */
+    @SuppressWarnings("Finally")
     static public float parseFloat(Object obj) {
         float val = Float.NaN;
         try {
             val = Float.parseFloat(obj.toString());
-        }  finally {
+        } finally {
             return val;
         }
```

Comment:
I don't think this is what you want to do.
IIRC the method will still fail if there's an exception.

If you just want to ignore the error, you want something like

```Java
        try {
            val = Float.parseFloat(obj.toString());
        } catch(Exception e) {
            // Ignore
        }
        return val;
```